docs/how_to/use_estimator.rst

```diff
@@ -0,0 +1,145 @@
+###########################
+Use the Estimator primitive
+###########################
+
+This guide shows how to get the expected value of an observable for a given quantum circuit with the :class:`~qiskit.primitives.Estimator` primitive.
+
+.. note::
+
+    While this guide only uses Qiskit Terra's implementation of the ``Estimator`` primitive, there are other
+    implementations of this primitive like Qiskit Terra's :class:`~qiskit.primitives.BackendEstimator`, Qiskit Aer's :class:`~qiskit_aer.primitives.Estimator`
+    and Qiskit Runtime's :class:`~qiskit_ibm_runtime.Estimator`.
```

Comment:
There is also this tutorial being done around selection Qiskit/qiskit-tutorials#1385 since you mention others here. That maybe helpful to link eventually to perhaps give clarity to the fact that there other others and how to select. Now the runtime has a bunch of tutorials for the runtime primitives too - would that be a "better" link than the API ref (Or in addition?)